lib/rules/object-curly-newline.js

```diff
@@ -139,6 +139,15 @@ module.exports = {
                 )
             );
 
+            // Use tokens or comments to check multiline or not.
+            // But use only tokens to check whether line breaks are needed.
+            // This allows:
+            //    var obj = { // eslint-disable-line foo
+            //        a: 1
+            //    }
```

Comment:
The ESLint style guide favors the following for multiline comments:

```js
/**
 * First line of comment text here
 * Second goes here, etc.
 * Last line
 */
```

With the leading slash aligned with the code block below.